.github/workflows/release-npm.yaml

```diff
@@ -0,0 +1,26 @@
+name: Publish Package to npmjs
```

Comment:
Can you instead integrate this into the existing release pipeline? https://github.com/facebook/dotslash/blob/main/.github/workflows/release.yml